client/styles/components/_form-container.scss

```diff
@@ -1,6 +1,7 @@
 .form-container {
 	text-align: center;
 	height: 100%;
+	min-height: #{556 / $base-font-size}rem;
```

Comment:
i imagine this min-height should be removed for desktop

Reply:
The min-width has no effect on desktop view (556 is default rendering height of the form), although I can add a media query , Should I?

Reply:
ah okay!  is this necessary because the parent is `height: 100%`? maybe there's a way to take that out?

Reply:
Maybe I could set the height property to 556 or remove the property and just leave min-width, would any of these be fine?

Reply:
i think it would be okay to just leave min-width!